tests/molecule/resources/monitoring/verify.yml

```
@@ -0,0 +1,40 @@
+- name: Monitoring verify
+  hosts: monitoring_server
```

```suggestion
  hosts: monitoring_host
```
I think it works at the minute because the containers are added to the `monitoring_server` group in the molecule configuration, but may be clearer to use the group define in the inventory

---

Thanks for picking that up!